src/components/missions/DisplayMissions.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import store from '../../redux/configureStore';
+import { removeMission } from '../../redux/missions/missionsSlice';
+
+const DisplayMissions = (props) => {
+  console.log(props);
```

Comment:
Kindly 🙏 remove all console log codes in your project. It is not good practice to leave the console log code in your project since it can expose 🤷‍♀️confidential data to a user😊

Reply:
Duly noted. Thanks